controllers/configurationpolicy_controller.go

```diff
@@ -335,15 +335,117 @@ func getTemplateConfigErrorMsg(err error) string {
 	return fmt.Sprintf("An unexpected error occurred when configuring the template resolver: %v", err)
 }
 
+type objectTemplateDetails struct {
+	kind         string
+	name         string
+	namespace    string
+	isNamespaced bool
+}
+
+func getDetails(templateObj *objectTemplateDetails, unstruct unstructured.Unstructured) {
```

Comment:
`getDetails` implies that the values are turned.
```suggestion
func addDetails(templateObj *objectTemplateDetails, unstruct unstructured.Unstructured) {
```